homeassistant/components/sensor/arlo.py

```diff
@@ -141,10 +189,12 @@ def device_state_attributes(self):
         attrs[ATTR_ATTRIBUTION] = CONF_ATTRIBUTION
         attrs['brand'] = DEFAULT_BRAND
 
-        if self._sensor_type == 'last_capture' or \
-           self._sensor_type == 'captured_today' or \
-           self._sensor_type == 'battery_level' or \
-           self._sensor_type == 'signal_strength':
+        known = False
+        for sensor_type in SENSOR_TYPES:
+            if self._sensor_type == sensor_type:
```

Comment:
We already validate this in the config schema. We can remove this check completely here.